addons/account/views/account_journal_dashboard_view.xml

```diff
@@ -20,13 +22,17 @@
                                 <div class="row">
                                     <t t-if="(journal_type == 'bank' || journal_type == 'cash')" t-call="JournalBodyBankCash"/>
                                     <t t-if="journal_type == 'sale' || journal_type == 'purchase'" t-call="JournalBodySalePurchase"/>
-                                    <t t-if="journal_type == 'general' || journal_type == 'situation'">
+                                    <t t-if="journal_type == 'situation'">
                                         <div class="col-6 o_kanban_primary_left">
                                             <button type="object" name="open_action" context="{'action_name': 'action_move_journal_line', 'view_no_maturity': 1}" class="btn btn-primary">
                                                 <span>Journal Entries</span>
                                             </button>
                                         </div>
                                     </t>
+                                    <t t-if="journal_type == 'general'">
+                                        <div class="col-12 o_kanban_primary_left o_general_activity">
```

Comment:
I don't like that placeholder...

the kanban box should always display the activities of the journal, and the widget should be defined in community